modules/actions/module.go

```diff
@@ -0,0 +1,61 @@
+package actions
+
+import (
+	"github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/forbole/juno/v3/modules"
+	"github.com/forbole/juno/v3/node"
+	"github.com/forbole/juno/v3/node/builder"
+	nodeconfig "github.com/forbole/juno/v3/node/config"
+	"github.com/forbole/juno/v3/types/config"
+
+	modulestypes "github.com/forbole/bdjuno/v2/modules/types"
+)
+
+const (
+	ModuleName = "actions"
+)
+
+var (
+	_ modules.Module                     = &Module{}
+	_ modules.AdditionalOperationsModule = &Module{}
+)
+
+type Module struct {
+	cfg     *Config
+	node    node.Node
+	sources *modulestypes.Sources
+}
+
+func NewModule(cfg config.Config, encodingConfig *params.EncodingConfig) *Module {
+	actionsCfg, err := ParseConfig(cfg.GetBytes())
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
do we want to hint users if `actions.port` is not set in `config.yaml` or it's unnecessary? for example:

```go
	if actionsCfg == nil {
		panic("hasura actions port needs to be specified in config.yaml")
	}
```